src/styles/main.scss

```diff
@@ -1,3 +1,16 @@
-body {
+@import "../utils/functions";
+@import "../utils/variables";
+@import "/blocks/card.scss";
+@import "/blocks/stars.scss";
+@import "/blocks/header.scss";
+@import "/blocks/showcase.scss";
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
 }
```

Comment:
don't add a folder for this file. it is good when it is on the same level as index.html